node-definitions/sonic/SONiCv.yaml

```diff
@@ -0,0 +1,61 @@
+id: SONiCv
+general:
+  nature: switch
+  read_only: false
+device:
+  interfaces:
+    has_loopback_zero: true
+    physical:
+      - Eth0
+      - Ethernet0
```

Comment:
More a question than a comment to change.  Why the increments of 4 for interface names?

Reply:
It comes from the number of lanes used by SONiC and the hardware platform